### DISCUSSION TITLE
Approval of WalletConnect registry: OneKey 🙏

### DISCUSSION THREAD

**Discussion by PatrickChoo** [WalletConnect/walletconnect-monorepo#1231] [Developer Support]
Hello, 

We have submitted our wallet "OneKey" in late May and It has been in PENDING REVIEW for a long time.
Could someone review the submission?

Thank you.

Website
https://onekey.so

Our Github Repo
https://github.com/OneKeyHQ/app-monorepo

Twitter
https://twitter.com/OneKeyHQ

Discord
https://discord.gg/onekey

**Comment by rayston92** [comment]
@PatrickChoo Can you also attach the submission link below for their reference?

**Reply by PatrickChoo** [reply]
I've updated the mobile deep link but it's still in PENDING REVIEW
